src/components/common/GitInfoMaterial.tsx

```diff
@@ -213,6 +220,13 @@ export default function GitInfoMaterial({
         toggleWebhookModal(selectedMaterial.id)
     }
 
+    const toggleExclude = (e): void => {
+        fromBulkCITrigger && stopPropagation(e)
+        const _showAllCommits = !showAllCommits
```

Comment:
why do we need this additional const?

Reply:
Because previously 'showAllCommits' was set as a constant

Reply:
done